src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -12,7 +12,8 @@
 
 import rbd
 
-from . import ApiController, RESTController, Task, UpdatePermission, DeletePermission
+from . import ApiController, RESTController, Task, UpdatePermission, \
+    DeletePermission, CreatePermission
```

Comment:
Sorry for the nitpicking but can you vertically align the start of DeletePermission with the start of ApiController?

Reply:
No problem! I was looking at an import in the same file "from ..services.exception" and imitating that style. While I'm at it, should I align that "serialize_dashboard_exception" with "handle_rados_error" too?

```
from . import ApiController, RESTController, Task, UpdatePermission, \
    DeletePermission, CreatePermission
from .. import mgr
from ..security import Scope
from ..services.ceph_service import CephService
from ..tools import ViewCache, str_to_bool
from ..services.exception import handle_rados_error, handle_rbd_error, \
    serialize_dashboard_exception
```

Reply:
yes, please :-)

Reply:
New try! Alignment made.